task/fbc-related-image-check/0.1/fbc-related-image-check.yaml

```diff
@@ -47,11 +50,13 @@ spec:
           fi
         done
         if [ -z "$FAILEDIMAGES" ]; then
-          HACBS_TEST_OUTPUT="$(make_result_json -r SUCCESS -s 1)"
+          note="Task $(context.task.name) succeeds, please check tekton task result for the details"
```

Comment:
suggested rewrite (SR): note="Task $(context.task.name) Success. Check Tekton task result for details."

Reply:
Yes, they can find the Tekton task result `HACBS_TEST_OUTPUT`, the name of result is added.